### PR TITLE
animation type spring

### PR DESCRIPTION
**Pull request by ErwinOtten**

### CODE REVIEW

File: packages/framer-scroller/hooks/useScrollTo.ts [hunk 1]
```diff
@@ -34,7 +34,9 @@ export function useScrollTo() {
               },
               onComplete,
               onStop: onComplete,
-              duration: duration * 1000 || distanceAnimationDuration(ref.scrollLeft, to.x),
+              type: 'spring',
+              bounce: 0,
+              duration: duration * 2000 || distanceAnimationDuration(ref.scrollLeft, to.x),
```
Comment (paales) [inline packages/framer-scroller/hooks/useScrollTo.ts:39]:
```suggestion
              duration: duration * 1000 || distanceAnimationDuration(ref.scrollLeft, to.x),
```

File: packages/framer-scroller/hooks/useScrollTo.ts [hunk 2]
```diff
@@ -54,7 +56,9 @@ export function useScrollTo() {
               },
               onComplete,
               onStop: onComplete,
-              duration: duration * 1000 || distanceAnimationDuration(ref.scrollTop, to.y),
+              type: 'spring',
+              bounce: 0,
+              duration: duration * 2000 || distanceAnimationDuration(ref.scrollTop, to.y),
```
Comment (paales) [inline packages/framer-scroller/hooks/useScrollTo.ts:61]:
```suggestion
              duration: duration * 1000 || distanceAnimationDuration(ref.scrollTop, to.y),
```

**Review by paales — CHANGES_REQUESTED** [review]

File: packages/framer-scroller/hooks/useScrollTo.ts (hunk 1 above)
Comment (ErwinOtten) [reply packages/framer-scroller/hooks/useScrollTo.ts:39]: All durations have to be adjusted if we go this route. Set it to 2000 for now to test.

Comment (paales) [reply packages/framer-scroller/hooks/useScrollTo.ts:39]: I dont think so, the duration is almost never provided and will almost always use the distanceAnimationDuration

**Pull request closed without merging**